sketches/M2_CAN_haz_bus/defines.h

```diff
@@ -0,0 +1,60 @@
+#ifndef __DEFINES_H__
```

Comment:
nice.  before we mainline this, let's please solidify the protocol, and update CAN_Haz_bus to use a defines header file as well.  whether this or a different one or both... we'll figure out.